### PR TITLE
refactor: use app name WHO COVID-19 everywhere

### PR DESCRIPTION
**Pull request by britannio**
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [x] REQUIRED: Do you have an Issue **assigned to you** within the v1 milestone for this PR?  Put the Issue number here: https://github.com/WorldHealthOrganization/app/issues/599
- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

## What does this PR accomplish?
closes https://github.com/WorldHealthOrganization/app/issues/599

### Notes
The applicationId on Android is still org.who.WHOMyHealth on android and the bundle id on IOS is still int.who.WHOMyHealth as changing these ids would most likely interfere with the release and they aren't displayed in the app.

## Did you add any dependencies?
no

## How did you test the change?
- Searching for occurrences of the string myHealth
- running the app on android and chrome and observing changes

### CODE REVIEW

**Comment by advayDev1** [comment]
Thanks!  cc @brunobowden, still up in the air re: short name vs long name

**Comment by PieterAelse** [comment]
What about the appName in the translation `commonWhoAppShareIconButtonDescription` ("Check out the official COVID-19 Guide App https://www.who.int/covid-19-app" ?

**Comment by britannio** [comment]
>What about the appName in the translation commonWhoAppShareIconButtonDescription?
Good spot, I can change the en translations, someone else needs to provide me the rest of the translations.

**Comment by britannio** [comment]
> Thanks! cc @brunobowden, still up in the air re: short name vs long name

@advayDev1 Has the name been confirmed?

Also sometimes the name appears as "...  Information App" and sometimes it's "... Guide App", is there a preference?

File: client/flutter/web/manifest.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
-    "name": "WHO MyHealth",
-    "short_name": "MyHealth",
+    "name": "WHO COVID-19",
+    "short_name": "WHO COVID-19",
```
Comment (advayDev1) [inline client/flutter/web/manifest.json]:
```suggestion
    "short_name": "COVID-19",
```

File: client/flutter/web/manifest.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
     "start_url": ".",
     "display": "minimal-ui",
     "background_color": "#0175C2",
     "theme_color": "#0175C2",
-    "description": "WHO MyHealth Web Preview",
+    "description": "WHO COVID-19 Web Preview",
```
Comment (advayDev1) [inline client/flutter/web/manifest.json]:
```suggestion
    "description": "WHO COVID-19 Web Preview [This is not approved content]",
```

File: client/flutter/web/index.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="UTF-8">
   <meta content="IE=Edge" http-equiv="X-UA-Compatible">
-  <meta name="description" content="WHO MyHealth Web Preview">
+  <meta name="description" content="WHO COVID-19 Web Preview">
```
Comment (advayDev1) [inline client/flutter/web/index.html]:
```suggestion
  <meta name="description" content="WHO COVID-19 Web Preview [This is not approved content]">
```

File: client/flutter/ios/Runner/Info.plist [hunk 4]
```diff
@@ -11,7 +11,7 @@
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
 	<key>CFBundleName</key>
-	<string>WHO MyHealth</string>
+	<string>WHO COVID-19</string>
```
Comment (advayDev1) [inline client/flutter/ios/Runner/Info.plist]:
```suggestion
	<string>WHO COVID-19</string>
	<key>CFBundleDisplayName</key>
	<string>COVID-19</string>
```

File: client/flutter/android/app/src/main/AndroidManifest.xml [hunk 5]
```diff
@@ -7,7 +7,7 @@
          FlutterApplication and put your custom class here. -->
     <application
         android:name="io.flutter.app.FlutterApplication"
-        android:label="WHO MyHealth"
+        android:label="WHO COVID-19"
```
Comment (advayDev1) [inline client/flutter/android/app/src/main/AndroidManifest.xml]:
```suggestion
        android:label="COVID-19"
```

File: client/flutter/web/index.html [hunk 6]
```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="UTF-8">
   <meta content="IE=Edge" http-equiv="X-UA-Compatible">
-  <meta name="description" content="WHO MyHealth Web Preview">
+  <meta name="description" content="WHO COVID-19 Web Preview [This is not approved content]">
```
Comment (brunobowden) [inline client/flutter/web/index.html]:
```suggestion
  <meta name="description" content="WHO COVID-19 Web Preview [TEST ONLY]">
```

**Pull request merged**